ystia/yorc
# Force purge if deployment status is not found when deployments are li…

# Pull Request description

## Description of the change
- Ensure purge is done if requested with undeploy
- Create taskTypeForcePurge to force deployment removal without any status check
- Force purge if deployment status is not found when deployments are listed and do not return it

### How to verify it
Deploy an app
Undeploy it with Yorc CLI without purge option (yorc d undeploy)
Remove deployment status in Consul
Check with "yorc d list" that the deployment is not listed and it has been removed from Consul

### Description for the changelog

## Applicable Issues
fixes #238
